azure-pipelines.yml

```diff
@@ -0,0 +1,42 @@
+pool:
+  vmImage: 'macos-10.14'
+
+variables:
+  scheme: 'CKB'
+  sdk: 'macosx10.14'
+  configuration: 'Debug'
+  skip_rpc_tests: 1
+  ckb_version: 'v0.13.0'
+  ckb_filename: 'ckb_$(CKB_VERSION)_x86_64-apple-darwin'
+
+steps:
+- task: CocoaPods@0
+  displayName: 'pod install using the CocoaPods task with defaults'
+  inputs:
+    forceRepoUpdate: true
+
+- task: Bash@3
+  inputs:
+    targetType: 'inline'
+    script: '
```

Comment:
Simpler to use YAML multiple line string

```
script: |
  line 1
  line 2
```

Reply:
Good to know. Changing to yml format for the whole file later.